tokio-trace/tokio-trace-core/src/metadata.rs

```diff
@@ -7,8 +7,21 @@ use std::fmt;
 
 /// Metadata describing a [span] or [event].
```

Comment:
The `[event]` link here is broken.

Reply:
Sorry, it is broken in `tokio-trace::Metadata`, not in `tokio-trace-core::Metadata`

Reply:
Related to that, is there any way for the two to be the same? Why are they distinct types with different docs (the docs for `tokio-trace::Metadata` haven't been updated to reflect the change below for example.

Reply:
They're the same type; `tokio-trace::Metadata` is a re-export of `tokio-trace-core::Metadata`. I think the issue is that `tokio-trace` depends on the crates.io version of `tokio-trace-core`, rather than the local version, so the changes I've made to the `-core` docs since the last crates.io release aren't reflected in the `tokio-trace` docs. 

I'll publish a `tokio-trace-core` point release before publishing `tokio-trace` 0.2, so I _think_ this should be resolved. In the meantime, you can patch `tokio-trace`'s dependency on `tokio-trace-core` to see the docs as they _should_ be.

Reply:
Ohh, I see. No worries, I just generated docs for `tokio-trace-core` explicitly. I think the `[event]` link is still broken though.

Reply:
I'll double check that one!